test/v2/models/response/retrieveHistoricFhlUkPiePeriodSummary/RentARoomSpec.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.response.retrieveHistoricFhlUkPiePeriodSummary
+
+import play.api.libs.json.{ JsValue, Json }
+import support.UnitSpec
+import v2.models.utils.JsonErrorValidators
+
+class RentARoomSpec extends UnitSpec with JsonErrorValidators {
+
+  val rentARoomIncome =RentARoomIncome(Some(5000.99))
+
+  val rentARoomExpenses = RentARoomExpenses(Some(5000.99))
+
+
+  val writesIncomeJson: JsValue = Json.parse(
+    """{
+      |      "rentsReceived":5000.99
+      |    }
+      |""".stripMargin
+  )
+
+  val readsIncomeJson: JsValue = Json.parse(
+    """{
+      |      "rentsReceived":5000.99
+      |    }
+      |""".stripMargin
+  )
+
+  val writesExpensesJson: JsValue = Json.parse(
+    """{
+      |      "amountClaimed":5000.99
+      |    }
+      |""".stripMargin
+  )
+
+  val readsExpensesJson: JsValue = Json.parse(
+    """{
+      |      "amountClaimed":5000.99
+      |    }
+      |""".stripMargin
+  )
+
+
+  "reads" when {
+    "passed a valid Income JSON" should {
+      "return a valid model" in {
```

Comment:
Please can these be "return a valid object" and "passed a valid Expenses object"? "Model" refers to the group of classes (definitions)

Reply:
Done